domains/asheer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "NullyIsHere",
+    "email": "",
```

Comment:
Email is neeed.

Reply:
https://github.com/is-a-dev/register/pull/14374
Stef told me email wasnt neccesary.